### PR TITLE
Bug fix on single player routing

### PR DESCRIPTION
**Pull request by ashtilawat**
Single Player Routing Bug Fix

What work was done? 
- Code was removed from index.js, so unnecessary components were not called. 
- This stopped the app from breaking. 
What was this significant?
-With this bug fix, single player mode is now working.

### CODE REVIEW

File: src/components/pages/Gamemode/Gamemode.js [hunk 1]
```diff
@@ -60,13 +60,7 @@ const Gamemode = ({ ...props }) => {
...
               <Header />
 
-              <Row>
-                <Col className="adventure-passport" xs={16} sm={24}>
-                  {!sP && props.child.gamemode.mode === 'select' && (
-                    <Link to="/gameification">{trig()}</Link>
-                  )}
-                </Col>
-              </Row>
+              <Row></Row>
```
Comment (MichaelMcFaddenjr) [inline src/components/pages/Gamemode/Gamemode.js:63]: Nice clean up!

**Review by artofmayhem — APPROVED** [review]
reviewed and done. thank you for the fix!

**Review by rhiannonstanford — APPROVED** [review]
Thank you for the code clean-up resolving the bug.

File: src/index.js [hunk 2]
```diff
@@ -41,7 +41,6 @@ import { NotFoundPage } from './components/pages/NotFound';
 import { ParentFaq } from './components/pages/ParentFaq';
 import { ParentContact } from './components/pages/ParentContact';
 import { NewParentDashboard } from './components/pages/NewParentDashboard';
-import { ParentDashFaq } from './components/pages/ParentDashFaq';
```
Comment (MichaelMcFaddenjr) [inline src/index.js:44]: nice catch removing these unused imports.

**Review by MichaelMcFaddenjr — APPROVED** [review]

**Review by milklot — APPROVED** [review]
Thanks for the fix, great job!

**Pull request merged**